modules/options-for-notebook-server-environments.adoc

```diff
@@ -157,13 +157,46 @@ a| * Python 3.9
 * Scikit-learn 1.2
 * SciPy 1.10
 
+.2+| HabanaAI
+| 2023.2 (Recommended)
+a| * Python 3.9
+* Habana 1.11
+* TensorFlow 2.12
+* Boto3 1.26
+* Kafka-Python 2.0
+* Matplotlib 3.6
+* Numpy 1.23
+* Pandas 1.5
+* Scikit-learn 1.2
+* SciPy 1.10
+* Kfp-tekton 1.5
+* PyTorch 2.0
+* Elyra 3.15
```

Comment:
Would be also nice to have all these sorted alphabetically. But that's an optional thing and probably should be done in a separate commit. I'll try to propose this in upstream of the project too.

Reply:
Sure Jan, because of time constraints, we can do this in a forthcoming release. I'd to create a separate Jira for this work, just to keep it separate from this.